addons/web/static/src/search/action_menus/action_menus.xml

```diff
@@ -7,7 +7,7 @@
                 <Dropdown items="asDropdownItems(printItems)">
                     <button class="btn btn-secondary" data-hotkey="shift+u">
                         <i class="fa fa-print me-1"/>
-                        <span class="o_dropdown_title">Print</span>
+                        <span class="o_dropdown_title" t-out="actionMenuTitle()"/>
```

Comment:
term `actionMenuTitle` doesn't really make sense here... Next to this dropdown, there's another one named `Actions`, I would think that that's the one for which I can set the custom name though this getter...